### ISSUE TITLE
Routes List showing errors

### ISSUE THREAD

**Issue by PabloCanalSuarez** [Arquisoft/viade_en3b1#48]
Some errors are shown to the user when:

- Parsing Turtle routes.
- Errors in JSON-LD routes.

**Comment by PabloCanalSuarez** [comment]
## Errors when trying to download a wrong JSON-LD route from POD
### Scenario
Trying to parse an input in JSON-LD that don't follow the ViadeSpec standard, like this `wrongRoute.jsonld`:
```json
{
    "@context": "http://schema.org",
    "@type": "Trip",
    "name": "1588247072_RutaTTLes1b",
    "itinerary": {
        "@type": "ItemList",
        "numberOfItems": 11,
        "itemListOrder": "http://schema.org/ItemListOrderDescending",
        "itemListElement": [
            {
                "@type": "GeoCoordinates",
                "latitude": 43.35595702623896,
                "longitude": -5.86845874786377
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.35814164005414,
                "longitude": -5.866398811340332
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.356331537050266,
                "longitude": -5.8631372451782235
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.35623790956414,
                "longitude": -5.8590173721313485
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.358797008861636,
                "longitude": -5.8590173721313485
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.36107514059645,
                "longitude": -5.85897445678711
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.361293587090316,
                "longitude": -5.866270065307617
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.358734593089714,
                "longitude": -5.869617462158204
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.35792318221192,
                "longitude": -5.870175361633301
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.3565500006228,
                "longitude": -5.8715057373046875
            },
            {
                "@type": "GeoCoordinates",
                "latitude": 43.356581209640375,
                "longitude": -5.875196456909181
            }
        ]
    },
    "media": []
}
```
### Expected result
The route is **ignored**, as it doesn't comply with ViadeSpec's JSON-LD standard.

### Result
Several **errors** raise:
![Captura de pantalla (267)](https://user-images.githubusercontent.com/56108450/80729248-42817100-8b08-11ea-9bf4-e4155192a846.png)

![Captura de pantalla (268)](https://user-images.githubusercontent.com/56108450/80728899-c71fbf80-8b07-11ea-844c-c166caf61cff.png)

**Comment by PabloCanalSuarez** [comment]
In principle, all these issues have been solved in f73f8aa.
